aries-site/src/layouts/content/UpdateNotification.js

```diff
@@ -0,0 +1,38 @@
+import { Anchor, Text, Notification } from 'grommet';
+import { ViewContext } from '../../pages/_app';
+import { useContext } from 'react';
+
+export const UpdateNotification = ({ name }) => {
+  const { contentHistory } = useContext(ViewContext);
+  const updateDate = contentHistory[name]?.date;
+
+  if (contentHistory && name in contentHistory) {
+    return (
+      <Notification
+        status="info"
+        margin={{ bottom: 'medium' }}
+        width="large"
+        message={
+          contentHistory[name]?.changeKind === 'Update' ? (
+            <Text>
+              {contentHistory[name]?.description + '    '}
+              {contentHistory[name]?.action?.length > 1 && (
+                <Anchor label="Link" href={contentHistory[name]?.action} />
+              )}
+            </Text>
+          ) : (
+            <Text>
+              This item is new. Let the Design System team know if you have any
+              feedback.
+            </Text>
+          )
+        }
+        title={`${
+          contentHistory[name]?.changeKind === 'Update'
+            ? 'Updated '
+            : 'Added on '
+        } ${new Date(updateDate).toDateString().split(' ').slice(1).join(' ')}`}
```

Comment:
Should we use one of these code snippets based on our guidance instead?

https://design-system.hpe.design/foundation/date-and-time?q=date#absolute-time

Reply:
Definitely, I went ahead and changed it to the "Long written format" code snippet